### DISCUSSION TITLE
Proposed experimental features to be made stable in 7.4

### DISCUSSION THREAD

**Discussion by JamesWTruher** [PowerShell/PowerShell#20214] [General]
As reviewed by the PS Committee,  the following experimental features are being considered to be made stable in the 7.4 release

- PSConstrainedAuditLogging
   - see  #19641
- PSCustomTableHeaderLabelDecoration
  - see #17346
- PSWindowsNativeCommandArgPassing
  - see #18706 although telemetry indicates that Windows mode is being set more than other modes. This maintains backwards compatibility.
- PSNativeCommandErrorActionPreference - the default value of the controlling variable would be set to $false for 7.4 and re-evaluated for 7.5
  - see #15897
- PSNativeCommandPreserveBytePipe
  - see #17857

Those experimental features not listed here are not being considered to be made stable.

**Comment by God-damnit-all** [comment]
Why didn't `PSCommandWithArgs` make the cut?

**Reply by SteveL-MSFT** [reply]
Need more data to show it's working for users' needs
